lib/util.js

```diff
@@ -208,6 +210,76 @@ const util = {
     fs.copySync(srcDir, dstDir)
   },
 
+  // In linux, WidevineCdm library is bundled instead of installing it by component
+  // updating during the runtime. In linux, we can't load cdm library during the runtime
+  // because processes except browser process are forked from zygote process not created from
+  // brave.exe binary. So, cdm should be loaded(not initialized) into zygote process space
+  // before zygote enters the sandbox.
+  // This method places WidevineCdm lib and header file into proper place before building.
```

Comment:
Sorry for delay in reviewing this, but for licensing reasons it's not allowed for us to bundle. 
This will need to happen on each client and we should use an npm config variable like`brave_google_api_endpoint`